packages/api-file-manager/src/plugins/context.ts

```diff
@@ -16,16 +16,13 @@ export default {
         // Get file storage plugin. We get it `byName` because we only support 1 storage plugin.
         const fileStoragePlugin = context.plugins.byName("api-file-manager-storage");
 
-        const settings = settingsCRUD(context);
-
         context.fileManager = {
             ...context.fileManager,
             files: filesCRUD(context),
             system: systemCRUD(context),
-            settings,
+            settings: settingsCRUD(context),
```

Comment:
We are sending `context`, and we also use the `fileStorageUpload` constant to pass the plugin to the `FileStorage` class. 

If that's the case, let's just get the plugin within the `FileStorage` class.